### ISSUE TITLE
Provide some useful feedback to the user in case of AccessDeniedException

### ISSUE THREAD

**Issue by ja422fs** [torakiki/pdfsam#255]
Log attached
[TaskIOException_on_Merge.txt](https://github.com/torakiki/pdfsam/files/1180145/TaskIOException_on_Merge.txt)

Windows 10
PDF Split and Merge Basic Edition
3.3.2
Java(TM) SE Runtime Environment 1.8.0_141-b15
Max memory 455 MB

**Comment by torakiki** [comment]
Hi, we are after this kind of exception for a while now... does it consistently happen or was it a one timer?

**Comment by ja422fs** [comment]
After initially installing PDFsam Basic, I got merge to work once or twice, but since then, merge fails every time.  I even tried uninstalling and reinstalling - no luck.

**Comment by torakiki** [comment]
Could you please make some test to see if it depends on the output folder? I see from the attached stacktrace that your output file was set into `C:\Program Files (x86)\PDFsam Basic`, could you please try few merges changing the output folder to see if that is the issue? Like `C:\Program Files (x86)\PDFsam Basic` always fails but some other folder never fails.
Do you have any AV or similar running?

**Comment by ja422fs** [comment]
I changed the output folder to be the same as that of the source files, and the merge worked.  I then retried the `C:\Program Files (x86)\PDFsam Basic`, and it failed as before.

I do have anti-virus running - Windows Defender.

**Comment by torakiki** [comment]
Ok, and when you used the failing path, did you type the output file path or did you select it by using the `Browse` button? From what we can see the native Windows file dialog should warn you about the missing permissions

**Comment by ja422fs** [comment]
When I used the failing output path, I typed the output file manually.  I hadn't used the Browse button.  However, I just tried using the Browse button to select the failing path (`C:\Program Files (x86)\PDFsam Basic`), and it gave me a warning that I don't have permission to save in this location.  It asked if I would like to save in the Documents folder instead.

![dialog](https://user-images.githubusercontent.com/30500603/28677936-ffdde2ae-72bc-11e7-8a47-90bdbc36f160.PNG)
